README.md

```diff
@@ -1,19 +1,251 @@
-# hazelcast-hibernate5
+# Hibernate Second Level Cache
 
-Hazelcast provides a distributed second level cache for your Hibernate entities, collections and queries.
+Hazelcast provides a distributed second-level cache for your Hibernate entities, collections, and queries.
+
+You can use an `IMap` as distributed storage(`HazelcastCacheRegionFactory`), or ConcurrentHashMap-based near-cache(`HazelcastLocalCacheRegionFactory`) with updates synchronized via `ITopic`.
```

Comment:
JCache is 3rd option here.

Reply:
Isn't JCache just an adapter that delegates the job to either `HazelcastCacheRegionFactory` or `HazelcastLocalCacheRegionFactory`?

Reply:
No. It is Hibernate's implementation for JCache Providers.